server/build/babel/find-config.js

```diff
@@ -1,16 +1,9 @@
 import { join } from 'path'
-import buildConfigChain from 'babel-core/lib/transformation/file/options/build-config-chain'
+import {loadPartialConfig} from '@babel/core'
 
 export default function findBabelConfig (dir) {
   // We need to provide a location of a filename inside the `dir`.
   // For the name of the file, we could be provide anything.
   const filename = join(dir, 'filename.js')
-  const options = { babelrc: true, filename }
-
-  // First We need to build the config chain.
-  // Then we need to remove the config item with the location as "base".
-  // That's the config we are passing as the "options" below
-  const configList = buildConfigChain(options).filter(i => i.loc !== 'base')
-
-  return configList[0]
+  return loadPartialConfig({ babelrc: true, filename })
```

Comment:
I'd recommend against passing `babelrc: true`, since you pass `filename` that is already the default, and it's possible we may change the meaning of `true` still in the beta to be broader than you'd actually want.